addons/pos_hr/models/hr_employee.py

```diff
@@ -21,6 +21,11 @@ def _load_pos_data_domain(self, data):
     def _load_pos_data_fields(self, config_id):
         return ['name', 'user_id', 'work_contact_id']
 
+    def _load_pos_data(self, data):
+        domain = self._load_pos_data_domain(data)
+        fields = self._load_pos_data_fields(data['pos.config'][0]['id'])
+        return self.search_read(domain, fields, load=False) if domain is not False else []
+
```

Comment:
```suggestion
    def _server_date_to_domain(self, domain):
        return domain
```

I'd prefer this solution. less duplicates.